apps/ledger-live-mobile/src/screens/ImportAccounts/DisplayResultItem.tsx

```diff
@@ -38,11 +44,15 @@ export default class DisplayResultItem extends Component<{
         </Flex>
         {!selectable ? null : (
           <Flex ml={8}>
-            <CheckBox
-              onChange={importing ? undefined : this.onSwitch}
-              isChecked={checked}
-              style={styles.marginLeft}
-            />
+            {importing ? (
+              <Importing />
```

Comment:
I would suggest importing InfiniteLoader from native-ui lib instead
```js
import { InfiniteLoader } from "@ledgerhq/native-ui";

...

<InfiniteLoader />
```

Reply:
ok, i'm updating the code, but i can't test it locally (my computer have issue building Android :'( )